neurovault/apps/statmaps/models.py

```diff
@@ -347,3 +351,33 @@ class Atlas(Image):
 
     class Meta:
         verbose_name_plural = "Atlases"
+
+
+class Similarity(models.Model):
```

Comment:
Cool - I had a second (closer) look at the implementation and I think it should be fine.